src/styles/global.css

```diff
@@ -71,6 +71,10 @@ html {
     border: none !important;
 }
 
+/* h1 title within page body */
+#main-column h1 { margin-left: 3rem; }
```

Comment:
Recommend removing the style and instead putting the h1 in the following:

<div class="container page-container">
  <div class="row site-content">
    h1 here
  </div>
</div>

That way in future, if the structure or margins of the page layout changes, the h1 still sits at the correct margin-left position.

Reply:
changes are complete